runtime/cfdumper/main.c

```diff
@@ -1042,6 +1042,8 @@ static void printMethod(J9CfrClassFile* classfile, J9CfrMethod* method)
 			j9tty_printf( PORTLIB, "long");
 			break;
 
+		case 'Q':
+			j9tty_printf( PORTLIB, "flattenable"); /* fallthrough */
```

Comment:
where you able to output a classfile with cfdump containing fields and methods with Q signatures?

Reply:
I was: both Q type fields and Q type methods were observed printing `flattenable Point2D` when appropriate